Curtains/Curtains.Api/Startup.cs

```diff
@@ -28,7 +28,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services
                 .AddApplication()
-                .AddInfrastructure(connectionString, _configuration);
+                .AddInfrastructure("Server=localhost,1433;Database=Curtains;User=sa;Password=<YourStrong@Passw0rd>", _configuration);
```

Comment:
Why string instead of connectionString variable?

Reply:
sorry my bad